craco.config.cjs

```diff
@@ -93,6 +94,16 @@ module.exports = {
       // See https://vanilla-extract.style/documentation/integrations/webpack/#identifiers for docs.
       // See https://github.com/vanilla-extract-css/vanilla-extract/issues/771#issuecomment-1249524366.
       new VanillaExtractPlugin({ identifiers: 'short' }),
+      new RetryChunkLoadPlugin({
+        cacheBust: `function() {
```

Comment:
FYI you actually can just define these functions in Javascript above, and call `.toString()` on them

function cacheBust() {
  return 'cache-bust=' + Date.now();
}

function retryDelay(retryAttempt) {
  return 2 ** (retryAttempt - 1) * 500;
}

// Use the functions and convert to strings
new RetryChunkLoadPlugin({
  cacheBust: cacheBust.toString(),
  retryDelay: retryDelay.toString(),
  maxRetries: 3,
});